dunstrc

```diff
@@ -144,6 +144,15 @@
     # Possible values are "start", "middle" and "end".
     ellipsize = middle
 
+    # Where to save notification the notification log
+    log_path = "~/.dunst.log"
+
+    # Log all notifications and save them in a FILE
+    log_notifications = no
```

Comment:
Oh, please make the defaults consistent to `config.h`.

Reply:
And why do you put the `log_path` as the **first** option, while there is a boolean `log_notifications`!?